### ISSUE TITLE
EMNLP-2019-Accepted Papers

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#255]
**Summary:**

This issue collects the papers I am interested in.

**Resource:**

- [paper list](https://www.emnlp-ijcnlp2019.org/program/accepted/)

**Next Reading:**

# News related

- Different Acquisition from the Same Sharing: Sifting Multi-task Learning for Fake News Detection
- Fine-Grained Analysis of Propaganda in News Articles
- Identifying Predictive Causal Factors from News Streams
- Learning to Update Knowledge Graph by Reading News
- News2vec: News Network Embedding with Subnode Information
- Nonsense!: Quality Control via Two-Step Reason Selection for Annotating Local Acceptability and Related Attributes in News Editorials
- Read, Attend and Comment: A Deep Architecture for Automatic News Comment Generation
- Countering the effects of lead bias in news summarization via multi-stage training and auxiliary losses
- Measure Country-Level Socio-Economic Indicators with Streaming News: An Empirical Study
- Neural News Recommendation with Multi-Head Self-Attention
- Visualizing Trend of Key Roles in News Articles

# Knowledge related
- A Knowledge Regularized Hierarchical Approach for Emotion Cause Analysis
- A Non-commutative Bilinear Model for Answering Path Queries in Knowledge Graphs
- Asking Clarification Questions in Knowledge-Based Question Answering
- CaRe: Open Knowledge Graph Embeddings
- Collaborative Policy Learning for Open Knowledge Graph Reasoning
- DIVINE: A Generative Adversarial Imitation Learning Framework for Knowledge Graph Reasoning
- DyKgChat: Benchmarking Dialogue Generation Grounding on Dynamic Knowledge Graphs
- Event Representation Learning Enhanced with External Commonsense Knowledge
- Fine-grained Knowledge Fusion for Sequence Labeling Domain Adaptation
- Generating Questions for Knowledge Bases via Incorporating Diversified Contexts and Answer-Aware Loss
- Improving Relation Extraction with Knowledge-attention
- Incorporating External Knowledge into Machine Reading for Generative Question Answering
- Incorporating Graph Attention Mechanism into Knowledge Graph Reasoning Based on Deep Reinforcement Learning
- Investigating BERT’s Knowledge of Language: Five Analysis Methods with NPIs
- KagNet: Learning to Answer Commonsense Questions with Knowledge-Aware Graph Networks
- Knowledge Aware Conversation Generation with Explainable Reasoning on Augmented Graphs
- Knowledge Enhanced Contextual Word Representations
- Knowledge-Enriched Transformer for Emotion Detection in Textual Conversations
- KnowledgeNet: A Benchmark Dataset for Knowledge Base Population
- Language Models as Knowledge Bases?
- Meta Relational Learning for Few-Shot Link Prediction in Knowledge Graphs
- Multi-task Learning for Conversational Question Answering Over a Large-Scale Knowledge Base
- Open Relation Extraction: Relational Knowledge Transfer from Supervised Data to Unsupervised Data
- Patient Knowledge Distillation for BERT Model Compression
- PullNet: Open Domain Question Answering with Iterative Retrieval on Knowledge Bases and Text
- Representation Learning with Ordered Relation Paths for Knowledge Graph Completion
- Semi-supervised Entity Alignment via Joint Knowledge Embedding Model and Cross-graph Model
- Tackling Long-Tailed Relations and Uncommon Entities in Knowledge Graph Completion
- Towards Knowledge-Based Recommender Dialog System
- Weakly Supervised Cross-lingual Semantic Relation Classification via Knowledge Distillation
- What’s Missing: A Knowledge Gap Guided Approach for Multi-hop Question Answering
- Adapting Meta Knowledge Graph Information for Multi-Hop Reasoning over Few-Shot Relations
- Commonsense Knowledge Mining from Pretrained Models
- Event Causality Recognition Exploiting Multiple Annotators’ Judgments and Background Knowledge
- Feature2Vec: Distributional semantic modelling of human property knowledge
- GlossBERT: BERT for Word Sense Disambiguation with Gloss Knowledge
- Incorporating Domain Knowledge into Medical NLI using Knowledge Graphs
- Leveraging Adjective-Noun Phrasing Knowledge for Comparison Relation Prediction in Text-to-SQL
- Rewarding Coreference Resolvers for Being Consistent with World Knowledge
- Using Pairwise Occurrence Information to Improve Knowledge Graph Completion on Large-Scale Datasets
- MAssistant: A Personal Knowledge Assistant for MOOC Learners

# RE related

- Connecting the Dots: Document-level Neural Relation Extraction with Edge-oriented Graphs
- Easy First Relation Extraction with Information Redundancy
- Improving Distantly-Supervised Relation Extraction with Joint Label Embedding
- Improving Relation Extraction with Knowledge-attention
- Joint Event and Temporal Relation Extraction with Shared Representations and Structured Prediction
- Leveraging 2-hop Distant Supervision from Table Entity Pairs for Relation Extraction
- Leveraging Dependency Forest for Neural Medical Relation Extraction
- Looking Beyond Label Noise: Shifted Label Distribution Matters in Distantly Supervised Relation Extraction
- **Open Relation Extraction: Relational Knowledge Transfer from Supervised Data to Unsupervised Data**
- Self-Attention Enhanced CNNs and Collaborative Curriculum Learning for Distantly Supervised Relation Extraction
- Cross-Sentence N-ary Relation Extraction using Lower-Arity Universal Schemas
- OpenNRE: An Open and Extensible Toolkit for Neural Relation Extraction
- Connecting the Dots: Document-level Neural Relation Extraction with Edge-oriented Graphs

# Others

- Doc2EDAG: An End-to-End Document-level Framework for Chinese Financial Event Extraction
-
